.gitignore

```diff
@@ -22,3 +22,7 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+#Ignore OSX DS_Store
+.DS_Store
```

Comment:
It's generally nicer to add this to your global gitignore. I can show you how tomorrow if you'd like.

Reply:
OK. Let's do that tomorrow